src/imagej/convert.py

```diff
@@ -510,12 +510,15 @@ def index_img_to_roi_manager(
         print("The RoiManager is unavailable in headless mode.")
 
 
-def _get_contours(ij: "jc.ImageJ", index_img: np.ndarray) -> List:
+def _get_contours(
+    ij: "jc.ImageJ", index_img: np.ndarray
+) -> List["jc.DefaultWritablePolygon2D"]:
```

Comment:
This should not be typed on the default implementation, but rather on the corresponding interface, `WritablePolygon2D`.

Reply:
Done! eaa7e97d961540247bb585d53606411bf1af4e0e